proxy/proxy.go

```diff
@@ -162,14 +162,20 @@ type PassiveHealthCheck struct {
 
 	// The maximum probability of unhealthy endpoint to be dropped out from load balancing for every specific request
 	MaxDropProbability float64
+
+	// The maximum ratio of unhealthy endpoints in the list of all endpoints PHC will check in case of all endpoints
+	// are unhealthy
+	MaxUnhealthyEndpointsRatio float64
 }
 
 func InitPassiveHealthChecker(o map[string]string) (bool, *PassiveHealthCheck, error) {
 	if len(o) == 0 {
 		return false, &PassiveHealthCheck{}, nil
 	}
 
-	result := &PassiveHealthCheck{}
+	result := &PassiveHealthCheck{
+		MaxUnhealthyEndpointsRatio: 1.0,
```

Comment:
Nice.

Could you also add another default optional explicitly (`MinDropProbability: 0.0`)?